### PR TITLE
add shipping methods to documentation for checkout.

### PR DESCRIPTION
**Pull request by Gregsen**
add shipping methods to documentation for checkout

### CODE REVIEW

**Comment by Gregsen** [comment]
👍

File: source/includes/_checkout-api.md [hunk 1]
```diff
@@ -768,9 +783,9 @@ X-Forwarded-For | Header | IP of the customer device issuing the request. If not
 OAuth token | Header | Customer token used to authenticate the request. The token also helps validate that the cart and addresses being used belong to the user making the request | Yes
 Checkout creation request object | Body | Input that clients can/need to provide for checkout creation | Yes
 
-## Changing the checkout
+## Updating the checkout
```
Comment (pc-alves) [inline source/includes/_checkout-api.md]: Nothing against using 'Updating', but if you're changing it to this then for consistency you should do it everywhere else.

**Comment by Gregsen** [comment]
:+1:

**Comment by pc-alves** [comment]
👍

**Pull request merged**